### PR TITLE
refactor: deprecate inline operator in favour of addField

### LINKED ISSUE #598: Deprecate `@inline` operator
List of tasks 

- [ ] Drop inline operator
- [ ] Fix tests which use inline with addField operator

### PR DESCRIPTION
**Pull request by ologbonowiwi**
- test: update tests/graphql
- refactor: drop inline field
- refactor: include `@addField` directive
- test: delete unnecessary test

**Summary:**  
_Briefly describe the changes made in this PR._
Dropped `@inline` directive from implementation and `tailcall.graphql`, added `@addField` directive on `tailcall.graphql`, updated `@inline` tests to achieve the same behavior using `@addField`

**Issue Reference(s):**  
Fixes #598
/claim #598

**Build & Testing:**

- [x] I ran `cargo test` successfully.
- [x] I have run `./lint.sh --mode=fix` to fix all linting issues raised by `./lint.sh --mode=check`.

**Checklist:**

- [x] I have added relevant unit & integration tests.
- [ ] I have updated the documentation accordingly (if applicable).
- [ ] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by ologbonowiwi** [comment]
Hey @tusharmath @sujeetsr @amitksingh1490, when I run `./lint.sh --mode=fix`, after formatting the files, I'm getting this error on the tests
image

Any guesses on how to fix it?

**Comment by tusharmath** [comment]
@ologbonowiwi you can add that file to `.prettierrc`

**Comment by ologbonowiwi** [comment]
Done; thank you!

File: tests/graphql/errors/test-const-with-inline.graphql [hunk 1]
```diff
@@ -3,8 +3,8 @@ schema @server @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
   query: Query
 }
 
-type Query {
-  post: Post @http(path: "/posts/1") @inline(path: ["user", "name"])
+type Query @addField(name: "username", path: ["user", "name"]) {
```
Comment (amitksingh1490) [inline tests/graphql/errors/test-const-with-inline.graphql:6]:
```suggestion
type Query @addField(name: "username", path: ["post","user", "name"]) {
```
Ideally this test should assert addfields and const in the path should give error

Comment (ologbonowiwi) [reply tests/graphql/errors/test-const-with-inline.graphql]: I didn't notice this; I fixed it! nice catch!

**Review by amitksingh1490 — APPROVED** [review]

**Comment by tusharmath** [comment]
@ologbonowiwi can you update the documentation for this change?

**Comment by ologbonowiwi** [comment]
Hey @tusharmath, what needs to be updated?

@sujeetsr already deleted `@inline` mentions and changed for `@addField` in [54c54b6](https://github.com/tailcallhq/tailcallhq.github.io/commit/54c54b6abd1de3072f7e2f0d5d036dfb6f7c2117).

If there's anything missing let me know.

**Pull request merged**